utils/getClaimStatus.tsx

```diff
@@ -7,66 +7,115 @@ import { ClaimStatusContent, I18nString, TextOptionalLink, TransLineProps } from
  */
-export function getClaimStatusSummary(scenarioType: ScenarioType): TransLineProps {
-  const transLineProps: TransLineProps = {
-    i18nKey: getTranslationPrefix(scenarioType) + '.summary.text',
+export function buildTransLineLinks(linkKeys: string[] | undefined): string[] {
+  const links: string[] = []
+  if (linkKeys) {
+    if (linkKeys && linkKeys.length > 0) {
+      for (const linkKey of linkKeys) {
+        // Explicitly cast to one of the allowed keys in urls.json
+        const key = linkKey as UrlType
+        const url = getUrl(key)
+        if (url) {
+          links.push(url)
+        }
+      }
+    }
   }
-  // Explicitly initialize to an empty array.
-  transLineProps.links = []
+  return links
+}
 
-  // Explicitly cast the scenarioType into one of the keys of claim-status.json
-  // (i.e. scenario1 | scenario2 etc)
-  const scenarioString = ScenarioType[scenarioType].toLowerCase() as keyof typeof claimStatusJson.scenarios
-  // Retrieve the summary links for the current scenario.
-  const summary = claimStatusJson.scenarios[scenarioString].summary as TextOptionalLink
-  const linkKeys = summary.links
+/**
+ * Build props to pass to the TransLine react component.
+ */
+export function buildTransLineProps(json: TextOptionalLink, i18nKey: I18nString): TransLineProps {
+  return {
+    i18nKey: i18nKey,
+    links: buildTransLineLinks(json.links),
+  }
+}
 
-  // Lookup each link and send the array to the TransLine component.
-  if (linkKeys && linkKeys.length > 0) {
-    for (const linkKey of linkKeys) {
-      // Explicitly cast to one of the allowed keys in urls.json
-      const key = linkKey as UrlType
-      const url = getUrl(key)
-      if (url) {
-        transLineProps.links.push(url)
-      }
-    }
+/**
+ * Helper function to construct the i18nKey for claim status link text.
+ */
+function buildI18nKey(keys: string[]): I18nString {
+  return 'claim-status:' + keys.join('.') + '.text'
+}
+
+/**
+ * Get Claim Status summary.
+ */
+export function buildClaimStatusSummary(
+  scenarioObject: ClaimStatusScenarioJson,
+  scenarioString: string,
+): TransLineProps {
+  const keys = ['scenarios', scenarioString, 'summary']
+  return buildTransLineProps(scenarioObject.summary, buildI18nKey(keys))
+}
+
+/**
+ * Get next steps content for Claim Status.
+ */
+export function buildNextSteps(
+  scenarioObject: ClaimStatusScenarioJson,
+  scenarioString: string,
+  whichSteps: StepType,
+): TransLineProps[] {
+  const steps: TransLineProps[] = []
+  const json = scenarioObject[whichSteps]
+  for (const [index, value] of json.entries()) {
+    const keys = ['scenarios', scenarioString, whichSteps, index.toString()]
+    steps.push(buildTransLineProps(value, buildI18nKey(keys)))
   }
-  return transLineProps
+  return steps
 }
 
 /**
  * Get combined Claim Status content.
  */
 export default function getClaimStatus(scenarioType: ScenarioType): ClaimStatusContent {
```

Comment:
@lomky I've renamed most of the functions in this file to `buildTheThing` but I left the default function for now to avoid renaming files and imports. We can do more renaming in #317